src/index.test.ts

```diff
@@ -141,57 +141,76 @@ describe('Logical', () => {
   });
 });
 
-describe('Custom Functions', () => {
-  test('can use functions as expressions', () => {
-    const unMockDate = mockDateHelper(new Date('2020-01-20T00:00:00.000Z'));
+describe("Custom Functions", () => {
+  test("can use functions as expressions", () => {
+    const unMockDate = mockDateHelper(new Date("2020-01-20T00:00:00.000Z"));
     const rulesFn = ruleFactory('DATEISO("10m")');
     expect(rulesFn({})).toMatchSnapshot();
     unMockDate();
   });
 
-  test('can invoke date functions', () => {
-    const unMockDate = mockDateHelper(new Date('2020-01-20T00:00:00.000Z'));
+  test("can invoke DATEISO function", () => {
+    const unMockDate = mockDateHelper(new Date("2020-01-20T00:00:00.000Z"));
     const rulesFn = ruleFactory(
       [
-        { if: '3 >= 1', then: 'inTenMinutes = DATEISO("10m")' },
-        { return: 'inTenMinutes' },
+        { if: "3 >= 1", then: 'inTenMinutes = DATEISO("10m")' },
+        { return: "inTenMinutes" },
       ],
       { trace: false }
     );
 
-    const input = { addToDate: '10m' };
+    const input = { addToDate: "10m" };
     const result = rulesFn(input);
 
     expect(result).toMatch(/.*\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}.*/);
     unMockDate();
   });
 
-  test('can use functional array helpers', () => {
+  test("can invoke DATE function", () => {
+    const rulesFn = ruleFactory(
+      [
+        'today = DATE(DATEISO("0d"))',
+        'tomorrow = DATE(DATEISO("1d"))',
+        {
+          if: "tomorrow > today",
+          then: "return tomorrow - today",
+          else: "return 0",
+        },
+      ],
+      { trace: false }
+    );
+    const result = rulesFn({});
+    // one millisecond difference locally vs CI
+    const isOneDay = result === 86400000 || result === 86400001;
```

Comment:
One day is 86400000 ms, but in CI the function is returning 86400001. This could be a problem, but it's above my pay grade.